zipkin4net/Criteo.Profiling.Tracing.Benchmark/Tracers/Zipkin/Spans.cs

```diff
@@ -26,8 +26,8 @@ private static Span CreateClientSpan()
             trace.Record(Annotations.Rpc("query"));
             trace.Record(Annotations.ServiceName("zipkin-client"));
             trace.Record(Annotations.Tag("jdbc.query", "select distinct `zipkin_spans`.`trace_id` from `zipkin_spans` join `zipkin_annotations` on (`zipkin_spans`.`trace_id` = `zipkin_annotations`.`trace_id` and `zipkin_spans`.`id` = `zipkin_annotations`.`span_id`) where (`zipkin_annotations`.`endpoint_service_name` = ? and `zipkin_spans`.`start_ts` between ? and ?) order by `zipkin_spans`.`start_ts` desc limit ?"));
-            trace.Record(Annotations.Tag("ca", "frontend"));
-            trace.Record(Annotations.Tag("sa", "backend"));
+            trace.Record(Annotations.ClientAddress("frontend"));
```

Comment:
so usually the client doesnt report itself. so for a client span, only the server address.

plus it likely would include the ip port info..

ex. I'd expect something like..
```
trace.Record(Annotations.ServerAddress("mysql", "172.19.0.2",  3306));
```

to end up in json like..

```json
    {
      "key": "sa",
      "value": true,
      "endpoint": {
        "serviceName": "mysql",
        "ipv4": "172.19.0.2",
        "port": 3306
      }
    }
```

fwiw in zipkin-js is it `ServerAddr({serviceName, host, port}) `